easybuild/easyconfigs/n/NVHPC/NVHPC-22.3-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,69 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'NVHPC'
+version = '22.3'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement
+# https://docs.nvidia.com/hpc-sdk/eula/index.html
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+local_tarball_tmpl = 'nvhpc_2022_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            'e0ea1cbb726556f6879f4b5dfe17238f8e7680c772368577945a85c0e08328f0',
+        local_tarball_tmpl % 'ppc64le':
+            '5e80db6010adc85fe799dac961ae69e43fdf18d35243666c96a70ecdb80bd280',
+        local_tarball_tmpl % 'x86_64':
+            'bc60a6faf2237bf20550718f71079a714563fa85df62c341cb833f70eb2fe7bb',
+    }
+]
+
+local_gccver = '11.2.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.37', '', ('GCCcore', local_gccver)),
```

Comment:
Same as in #15155: `GCC` should be used as toolchain to ensure that this dependency is reflected in the module name?